awesomerc.lua

```diff
@@ -175,6 +174,14 @@ awful.screen.connect_for_each_screen(function(s)
         end
         gears.wallpaper.maximized(wallpaper, s, true)
     end
+end
+
+-- Re-set wallpaper when a screen's geometry changes (e.g. different resolution)
+screen.connect_signal("property::geometry", set_wallpaper)
```

Comment:
@Elv13 
Do you mean to have a generic wrapper to connect to different signals then?  Do you have an example?
In this case I think it is clearer to specify it explicitly like it's done here.